pilot/pkg/kube/inject/testdata/webhook/hello-multi.yaml.injected

```diff
@@ -130,8 +131,8 @@ spec:
           optional: true
           secretName: istio.default
 status: {}
----
-apiVersion: extensions/v1beta1
+
+---apiVersion: extensions/v1beta1
```

Comment:
having the `---` in the same line as the next yaml object seems against yaml standards

Reply:
Good catch. Will fix it.